packages/story-editor/src/components/library/panes/text/karma/textSets.cuj.karma.js

```diff
@@ -226,12 +226,11 @@ describe('CUJ: Text Sets (Text and Shape Combinations): Using Text Sets', () =>
     it('should add text color based on background', async () => {
       await fixture.editor.library.text.smartColorToggle.click();
 
+      await fixture.events.sleep(500);
```

Comment:
🤔  are both sleeps here really needed? 

@BrookeGraham did a big chunk of work last week to remove redundant waitFor/sleeping, the new pattern looks like the below and is based on this: https://testing-library.com/docs/dom-testing-library/api-async/ 

```
await waitFor(() => {
const node = fixture.editor.canvas.quickActionMenu.changeBackgroundColorButton;
if (!node) {
throw new Error('this will make it so that the waitFor polls until no error is thrown');
}
expect(node).toBeTruthy();
} 

 await fixture.events.click(fixture.editor.canvas.quickActionMenu.changeBackgroundColorButton);
```

Reply:
Thanks, I will check that once I find the flaky part.